src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderInput.tsx

```diff
@@ -238,26 +241,30 @@ export const LimitOrderInput = ({
     isFetching: isLimitOrderQuoteFetching,
   } = useQuoteLimitOrderQuery(limitOrderQuoteParams)
 
+  useFindByAssetIdQuery(sellAsset.assetId, {
+    pollingInterval: MARKET_DATA_POLLING_INTERVAL,
+  })
+
+  useFindByAssetIdQuery(buyAsset.assetId, {
+    pollingInterval: MARKET_DATA_POLLING_INTERVAL,
+  })
```

Comment:
Not related to the PR but the naming `useFindByAssetIdQuery` isn't very descriptive, I can't determine what is it doing by reading those queries

Reply:
Actually related! 

Very valid point and I don't think we should change the naming in the slice (we usually use sliceName.endpoints.endpointName which already contains the domain, and more rarely queries) but rather it's probably a good idea to re-expose the queries with the name of the domain to make things more obvious at first read.

Reply:
Yes I was thinking about renaming the export instead of the slice of course, the endpoint name seems valid considering you are reading the slice scope, but outside it would make sense to rename the query hook to a more readable name

Reply:
gm? c8d92f97cedbf601dfa5d64eab3e00848791ac84

Reply:
purfect